src/NotificationServices/Softeq.NetKit.Services.SmsNotifications/SmsSender/TwilioHttpClient.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System.Threading.Tasks;
+using Twilio.Clients;
+using Twilio.Http;
+
+namespace Softeq.NetKit.Services.SmsNotifications.SmsSender
+{
+    public class TwilioHttpClient : ITwilioRestClient
+    {
+        private readonly ITwilioRestClient _innerClient;
+
+        public TwilioHttpClient(System.Net.Http.HttpClient httpClient, TwilioSmsConfiguration twilioSmsConfiguration)
+        {
+            _innerClient = new TwilioRestClient(
+                twilioSmsConfiguration.AccountSid,
+                twilioSmsConfiguration.AuthToken,
+                httpClient: new SystemNetHttpClient(httpClient));
```

Comment:
Can we use here HttpClient instead of SystemNetHttpClient class?

Reply:
no, Twilio.Http.HttpClient is an abstract class, so we should use Twilio.Http.SystemNetHttpClient which is inherited from HttpClient